libraries/abstractions/secure_sockets/utest/CMakeLists.txt

```diff
@@ -11,15 +11,16 @@
                 "${kernel_dir}/include/task.h"
                 "${kernel_dir}/include/portable.h"
                 "${common_dir}/include/iot_logging_task.h"
-                "${abstraction_dir}/wifi/include/iot_wifi.h"
                 "${freertos_plus_dir}/standard/tls/include/iot_tls.h"
                 "${3rdparty_dir}/lwip/src/include/lwip/sockets.h"
+                "${3rdparty_dir}/lwip/src/include/lwip/ip_addr.h"
+                "${3rdparty_dir}/lwip/src/include/lwip/dns.h"
             )
 # list the directories your mocks need
     list(APPEND mock_include_list
+		"${CMAKE_CURRENT_LIST_DIR}/include"
```

Comment:
Can you update this to use spaces instead of tabs so it's consistent with the rest of the file?

Reply:
Please make this change across all of the files as well.

Reply:
Done